perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -1055,16 +1055,30 @@ sub waste_munge_bulky_data {
     my %items = map { $_->{name} => $_->{bartec_id} } @items_list;
 
```

Comment:
I think `collection_date` (not in this PR I realise), might need to be `Collection_Date`, think other Echo code assumes it's that (as does the test).